### PR TITLE
Re-organized requirements files

### PR DESCRIPTION
**Pull request by clintonb**
The base requirements no longer make sense given the variety of scripts with individual requirements. Additionally, the Sailthru scripts have been moved to a parent package.

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/ecommerce please review
@schenedx where are the Sailthru Jenkins jobs that will need to be updated?

File: google_analytics/ga_homepage_report.py [hunk 1]
```diff
@@ -302,7 +305,8 @@ def clicksDataframe(clicks_data):
     clicks_dataframe = DataFrame(clicks_data, columns=['date', 'cardName', 'position', 'totalClicks', 'uniqueClicks'])
     clicks_dataframe = clicks_dataframe.apply(to_numeric, errors='ignore')
     clicks_dataframe.drop('date', axis=1, inplace=True)
-    clicks_dataframe = clicks_dataframe.groupby(['cardName','position']).sum().sort_values(by='uniqueClicks',ascending=0)
+    clicks_dataframe = clicks_dataframe.groupby(['cardName', 'position']).sum().sort_values(by='uniqueClicks',
```
Comment (amangano-edx) [inline google_analytics/ga_homepage_report.py]: As an alternative, we could shorten this line by moving the groupby arg into a named variable on the line above.
```
group_cols = ['cardName', 'position']
clicks_dataframe = clicks_dataframe.groupby(group_cols).sum().sort_values(by='uniqueClicks',ascending=0)
```

**Comment by schenedx** [comment]
@clintonb The PR for creating that jenkins job is https://github.com/edx/jenkins-job-dsl/pull/86
@umar-nawaz Please be-aware of this change. Maybe we should only merge your PR for Jenkins DSL only after this is merged.

File: google_analytics/ga_homepage_report.py [hunk 2]
```diff
@@ -322,7 +326,8 @@ def enrollmentDataframe(enrolls_data, card_type, enroll_type):
 
     enrolls_dataframe = enrolls_dataframe.apply(to_numeric, errors='ignore')
     enrolls_dataframe.drop('date', axis=1, inplace=True)
-    enrolls_dataframe = enrolls_dataframe.groupby(['cardName','position']).sum().sort_values(by='unique{type}Enrolls'.format(type=enroll_type),ascending=0)
+    enrolls_dataframe = enrolls_dataframe.groupby(['cardName', 'position']).sum().sort_values(
+        by='unique{type}Enrolls'.format(type=enroll_type), ascending=0)
```
Comment (amangano-edx) [inline google_analytics/ga_homepage_report.py]: Alternatively, we could move the groupby and sort_by args into named variables.
```
group_cols = ['cardName', 'position']
sort_by = 'unique{type}Enrolls'.format(type=enroll_type), ascending=0)
enrolls_dataframe = enrolls_dataframe.groupby(group_cols).sum().sort_values(by=sort_by, ascending=0)

```

File: google_analytics/ga_homepage_report.py [hunk 3]
```diff
@@ -444,7 +454,8 @@ def output_report(filename, total_homepage_views=None, total_course_card_clicks=
         homepage_subject_worksheet.set_column('C:S', 15, percent_fmt)
 
         # Write heading
-        homepage_subject_worksheet.write('A1', 'Top Subject Pages from the Homepage, Data from '+str(start_date)+' to '+str(end_date) , cell_format)
+        homepage_subject_worksheet.write('A1', 'Top Subject Pages from the Homepage, Data from ' + str(
```
Comment (amangano-edx) [inline google_analytics/ga_homepage_report.py]: Alternatively, we can construct the string in a named variable before passing it to this method.
```
# Note: a different variable name might be more appropriate, as I'm not familiar with this method call.
heading = 'Top Subject Pages from the Homepage, Data from ' + str(start_date) + ' to ' + str(end_date)
homepage_subject_worksheet.write('A1', heading, cell_format)
```

File: google_analytics/ga_homepage_report.py [hunk 4]
```diff
@@ -421,13 +426,18 @@ def output_report(filename, total_homepage_views=None, total_course_card_clicks=
...
-        featured_cards_worksheet.write('C13', float(featured_cards['uniqueEnrolls'].sum())/total_homepage_views, percent_fmt)
-        featured_cards_worksheet.write('C14', float(featured_cards['uniqueCourseEnrolls'].sum()) / total_homepage_views, percent_fmt)
-        featured_cards_worksheet.write('C15', float(featured_cards['uniqueProgramEnrolls'].sum()) / total_homepage_views, percent_fmt)
+        featured_cards_worksheet.write('C13', float(featured_cards['uniqueEnrolls'].sum()) / total_homepage_views,
+                                       percent_fmt)
+        featured_cards_worksheet.write('C14', float(featured_cards['uniqueCourseEnrolls'].sum()) / total_homepage_views,
+                                       percent_fmt)
+        featured_cards_worksheet.write('C15',
+                                       float(featured_cards['uniqueProgramEnrolls'].sum()) / total_homepage_views,
+                                       percent_fmt)
```
Comment (amangano-edx) [inline google_analytics/ga_homepage_report.py]: Would it make sense to do some of these calculations above the method invocation and pass them as named variables instead?

File: google_analytics/ga_homepage_report.py (hunk 1 above)
Comment (clintonb) [inline google_analytics/ga_homepage_report.py]: It is not my intention to go through the actual code here. I got a bit overzealous with my IDE automatically cleaning code. This commit is solely intended to fix requirements and imports. I'll revert these spacing changes.

**Pull request merged**